stock_inventory_valuation_report/reports/stock_inventory_valuation_report.py

```diff
@@ -44,52 +44,48 @@ class StockInventoryValuationReport(models.TransientModel):
         help="Use compute fields, so there is nothing store in database",
     )
 
-    @api.multi
     def _compute_results(self):
         self.ensure_one()
         if not self.compute_at_date:
             self.date = fields.Datetime.now()
         products = (
             self.env["product.product"]
             .with_context(
-                dict(to_date=self.date, company_owned=True, create=False, edit=False)
+                to_date=self.date, company_owned=True, create=False, edit=False
             )
             .search([("type", "=", "product"), ("qty_available", "!=", 0)])
         )
         ReportLine = self.env["stock.inventory.valuation.view"]
         for product in products:
             standard_price = product.standard_price
-            if self.date:
-                standard_price = product.get_history_price(
-                    self.env.user.company_id.id, date=self.date
-                )
             line = {
                 "name": product.name,
                 "reference": product.default_code,
                 "barcode": product.barcode,
-                "qty_at_date": product.qty_at_date,
+                "qty_at_date": product.qty_available,
                 "uom_id": product.uom_id,
                 "currency_id": product.currency_id,
                 "cost_currency_id": product.cost_currency_id,
                 "standard_price": standard_price,
-                "stock_value": product.qty_at_date * standard_price,
+                "stock_value": product.qty_available * standard_price,
                 "cost_method": product.cost_method,
             }
-            if product.qty_at_date != 0:
+            if product.qty_available != 0:
```

Comment:
@solo4games Shouldn't it be 'quantity_svl' field instead ?

Reply:
I am not sure about that, because, when I did it, one of the tests were failed because it did not pass that if condition, so I can not said that I am right

Reply:
This condition should be dropped as it is already part of the search on products